doc/modules/ROOT/pages/end-to-end-examples/load-data-via-graph-construction.adoc

```diff
@@ -0,0 +1,167 @@
+= Load data to a projected graph via graph construction
+
+[TIP]
+====
+Jupyter Notebook available on https://github.com/neo4j/graph-data-science-client/blob/main/examples/load-data-via-graph-construction.ipynb[Github^]
+====
+
+This notebook shows the usage of the `gds.alpha.graph.construct` method
+(available only in GDS 2.1+) to build a graph directly in memory.
+
+*NOTE:* If you are using AuraDS, it is currently not possible to write
+the projected graph back to Neo4j.
+
+== Setup
+
+We need an environment where Neo4j and GDS are available, for example
+AuraDS (which comes with GDS preinstalled) or Neo4j Desktop.
+
+Once the credentials to this environment are available, we can install
+the `graphdatascience` package and create the `gds` object.
+
+[source, python]
```

Comment:
General comment: If we don't make it testable (indeed this may be too much work atm) we need to add `role=no-test` to the python source blocks